cmd/cloud/server.go

```diff
@@ -183,7 +189,7 @@ var serverCmd = &cobra.Command{
 			multiDoer = append(multiDoer, supervisor.NewGroupSupervisor(sqlStore, instanceID, logger))
 		}
 		if installationSupervisor {
-			multiDoer = append(multiDoer, supervisor.NewInstallationSupervisor(sqlStore, kopsProvisioner, awsClient, instanceID, clusterResourceThreshold, keepDatabaseData, keepFilestoreData, resourceUtil, logger))
+			multiDoer = append(multiDoer, supervisor.NewInstallationSupervisor(sqlStore, kopsProvisioner, awsClient, instanceID, clusterResourceThreshold, clusterResourceThresholdScaleValue, keepDatabaseData, keepFilestoreData, resourceUtil, logger))
```

Comment:
This is not a suggestion for this PR but we should probably think about refactoring `NewInstallationSupervisor` to take a struct just because 10 positional arguments is a bit unwieldy lol

Reply:
Totally agree. The two server threshold flags should eventually be moved to be part of the cluster object, but this allows us to see if we like the scaling or not before we do so.

I will create a ticket to tackle this later.

Reply:
Perfect, thanks @gabrieljackson